src/Core/Checkout/Customer/Validation/Constraint/CustomerEmailUnique.php

```diff
@@ -31,11 +31,11 @@ class CustomerEmailUnique extends Constraint
     public function __construct(array $options)
     {
         if (!($options['context'] ?? null) instanceof Context) {
-            throw new MissingOptionsException(\sprintf('Option "context" must be given for constraint %s', self::class), ['context']);
```

Comment:
did you make sure, that this exceptions does not have a specific purpose? at it is a Symfony exception

Reply:
As far as I can tell it's not being listened to anywhere specifically (in symfony or shopware)